website/src/word-panel.tsx

```diff
@@ -129,6 +126,28 @@ export const WordPanel = (p: {
   }
 }
 
+export const PartLines = (p: {
+  segments: Dailp.FormFieldsFragment["segments"]
+}) => {
+  if (p.segments) {
+    let length = p.segments.length
```

Comment:
Just trying to instill some better habits, let's call this `segmentCount` instead. And on line 136 instead of `part` lets call that `segment`, just to make all of the variable names consistent.